### ISSUE TITLE
About Dartmouth Lung Cancer Histology Dataset

### ISSUE THREAD

**Issue by chenqz1998** [BMIRDS/deepslide#37]
I downloaded the dataset but failed to find out the annotations. There is only whole-slide image (WSI) without annotations. How could I get the annotations made by pathologists?

**Comment by GeorgeBatch** [comment]
The WSI-level annotations are in the `MetaData_Release_1.0.csv` file. The `Class` column specifies WSI's predominant pattern.

**Comment by JosephDiPalma** [comment]
@GeorgeBatch You are correct that the wsi-level annotations are in the csv file.

@chenqz1998 The pathologists did not provide annotations for the test set, but rather a slide-level class. We only have annotations for the training and validation sets, but we cannot make those datasets public at this time.

**Comment by GeorgeBatch** [comment]
@JosephDiPalma Do you think you can make the train and validation sets available on a per-request basis like the test set? Maybe it can include more questions, or even an official request from a research group so there is more accountability in terms of what happens to the data?

**Comment by JosephDiPalma** [comment]
@GeorgeBatch I don't know but I can definitely check. It would depend on the privacy restrictions put forth by our institution.

**Comment by GeorgeBatch** [comment]
@JosephDiPalma Thank you very much! It will be very helpful! Please let me know when you have an answer.

**Comment by chenqz1998** [comment]
I see. Yes, I know there is WSI-level class annotation provided in `MetaData_Release_1.0.csv` file. But I wanna know the **area annotation** (e.g. the area provided in **Fig.4** in the paper "Pathologist-level classification of histologic patterns on resected lung adenocarcinoma slides with deep neural networks") to train and tese the model for detection.

**Comment by ntomita** [comment]
@chenqz1998 The ROI annotations presented in the figure are only available for the selected slides in the test set as it is meant for visualization and error analysis; thus it is not part of the dataset.
